### ISSUE TITLE
Radarr/Sonarr API Key Blank

### ISSUE THREAD

**Issue by astlaurent** [mdhiggins/sickbeard_mp4_automator#1282]
Hello

I just cloned this repo after using an old version for a long time. I set it back up on a Windows 10 system running Python3. When post processing a Radarr and Sonarr download I am getting this error in the Sab log. What is strange if the config file has a Radarr and Sonarr API key in it so i am not sure why it is saying it cannot be blank. 

020-07-13 14:31:50 - autoprocess.radarr - INFO - Radarr notifier started.
2020-07-13 14:31:50 - autoprocess.radarr - ERROR - Your Radarr API Key can not be blank. Update autoProcess.ini.

The Sonarr error is the same but stating the Sonarr API key cannot be blank

**Comment by mdhiggins** [comment]
Config file is located in the config directory not the root anymore. Make sure you're using the right config file

**Comment by astlaurent** [comment]
yup i have the config file in the config directory. I have the whole repo in a new folder and built the config file out from scratch using your .sample file.

**Comment by mdhiggins** [comment]
Gonna need to post your config file then

**Comment by astlaurent** [comment]
Hi, I actually figured out what the issue was. I did the configuration from the sample file that was in the setup directory but left it there so it was using the default one that was sitting in the config directory. I moved over the one i built and it is working now. Sorry about that and thanks for the help. 

One more question as it relates to the config file. On my old setup i was using Intel Quicksync. on my old config file i would set it up with this line
video-codec = h264qsv,h264,x264
Do i set it up the same on the new configuration?
